autonomy/cli/deploy.py

```diff
@@ -195,58 +250,45 @@ def build_deployment(  # pylint: disable=too-many-arguments
 @click.option("--push", is_flag=True, default=False, help="Push image after build.")
 @image_profile_flag()
 def build_images(  # pylint: disable=too-many-arguments
-    service_id: PublicId,
     profile: str,
-    packages_dir: Path,
-    build_dir: Path,
-    skaffold_dir: Path,
+    packages_dir: Optional[Path],
+    build_dir: Optional[Path],
+    skaffold_dir: Optional[Path],
     version: str,
     push: bool,
 ) -> None:
     """Build image using skaffold."""
 
+    if build_dir is None:
+        build_dir = Path.cwd()
+
+    if packages_dir is None:
+        packages_dir = Path.cwd() / PACKAGES
+
+    if skaffold_dir is None:
+        skaffold_dir = Path.cwd() / "Dockerfiles"
+
```

Comment:
this looks problematic to me. Are these variables independent? Then you can stop reading here, otherwise imagine the following example:

we provide `build_dir` but not the `packages_dir` or `skaffold_dir`. Or _vice versa_ for that matter.  Now we have one as current working dir and the other in the specified location. Therefore I would expect something like:

```
build_dir = build_dir or Path.cwd()
packages_dir = packages_dir or (build_dir / PACKAGES)
skaffold_dir = skaffold_dir or (build_dir / "Dockerfiles")
```

also good to declare `"Dockerfiles"` as a constant

Reply:
I already did it https://github.com/valory-xyz/open-autonomy/blob/feat/port-deployments-folder/autonomy/deploy/constants.py#L112 I'll fix this